nat/traversal/nat_proxy.go

```diff
@@ -43,7 +42,6 @@ func newNATProxy() *natProxy {
 }
 
 func (np *natProxy) consumerHandOff(consumerAddr string, remoteConn *net.UDPConn) chan struct{} {
-	time.Sleep(400 * time.Millisecond)
```

Comment:
As commit message states, this was introduced to workaround the case with very slow providers. I tested it on some weak machine with a single core.

Reply:
We have now whole ping `Timeout` in 10 seconds to finish it. This should not be a problem now.

Reply:
It would be good to avoid such random sleeps and redesign code. You never how weak machine is :D